nuxt.config.ts

```diff
@@ -36,6 +36,7 @@ export default defineNuxtConfig({
       ALTERNATIVE_NETWORK_NAME: process.env.ALTERNATIVE_NETWORK_NAME,
       AE_TOKEN_ID: process.env.AE_TOKEN_ID,
       DEBUG_MODE: process.env.DEBUG_MODE,
+      SH_CONTRACTS: process.env.SH_CONTRACTS?.split(';'),
```

Comment:
I would call this `SH_DEX_CONTRACTS` to make it more explicit.

Reply:
Okay, good. I renamed that and asked Petar to rename the variable for deployment environments. So maybe for some short time period, it will not be working. I will inform you about the current status

Reply:
Seems to be good now